Routers/Routers/Prima.cs

```diff
@@ -0,0 +1,121 @@
+using Microsoft.VisualBasic;
+
+public class Graphe
```

Comment:
```suggestion
public class Graph
```
правильно писать Graph